datafusion/physical-expr/Cargo.toml

```diff
@@ -35,12 +35,15 @@ path = "src/lib.rs"
 [features]
 crypto_expressions = ["md-5", "sha2", "blake2", "blake3"]
 default = ["crypto_expressions", "regex_expressions", "unicode_expressions"]
```

Comment:
I think we should keep dictionary support as a default, if possible

Reply:
I fairly strongly disagree, it is pretty esoteric. As a data point, none of IOx's integration tests require this, and we use dictionaries a LOT :smile:

Reply:
> I fairly strongly disagree, it is pretty esoteric. 

I am not sure how to quantify how esoteric the feature is or how commonly it is used. Clearly IOx uses it. I was just thinking that this PR changes the default behavior

But maybe that is ok.

Perhaps some other committers have thoughts.

Reply:
> Clearly IOx uses it

That is the point I'm trying to make, IOx **doesn't** use it, at least not within any of its tests

Reply:
> at least not within any of its tests.

🤔  when I didn't enable the dyn dictionary kernels in arrow iox tests failed in some past version

We have it enabled here: https://github.com/influxdata/influxdb_iox/blob/6f39ae342e64848bd6555bddbc1d3fa30050f75e/arrow_util/Cargo.toml#L12